string-manipulation-2/src/last-chars.js

```diff
@@ -0,0 +1,25 @@
+/* exported lastChars */
+
+// check if length is greater than string length...
+// if yes, return the full original string
+// check if the length is 1...
+// if yes, return the last letter of the string
+// declare variable to hold final string
+// declare variable for the starting index which is the string length minus given length
+// loop over the string starting at starting index...
+// concatenate the letters to the final string
+// return the final string
+
+function lastChars(length, string) {
+  if (length > string.length) {
+    return string;
+  } else if (length === 1) {
+    return string[string.length - 1];
+  }
```

Comment:
I don't think this conditional block is necessary 🤔

Reply:
I find myself adding unnecessary steps when I go back over the code to check for special cases... thank you!